server/src/test/scala/services/graphstages/ArrivalsGraphStagePaxNosSpec.scala

```diff
@@ -28,47 +29,47 @@ class ArrivalsGraphStagePaxNosSpec extends CrunchTestLike {
     }
 
     "When I send an ACL arrival with zero pax" >> {
-      offerArrivalAndWait(crunch.aclArrivalsInput, scheduled = nowString, actPax = Option(0), tranPax = None, maxPax = Option(150))
+      offerArrivalAndWait(crunch.aclArrivalsInput ,scheduled = nowString, actPax = Option(0), tranPax = None, maxPax = Option(150))
```

Comment:
Can we fix the formatting. A lot of commas in the wrong place in this file. ctrl-alt-L fixes these automatically in IntelliJ - though clearly different on the mac